Tests/js/views/assets/ez-navigationhubview-tests.js

```diff
@@ -3,7 +3,7 @@
  * For full copyright and license information view LICENSE file distributed with this source code.
  */
 YUI.add('ez-navigationhubview-tests', function (Y) {
-    var viewTest, eventTest, logOutTest,
+    var viewTest, eventTest,
```

Comment:
the `userChange` event handler appears as covered but it's not really tested and the `userAvatarChange` is not covered at all.